labelme/config/default_config.yaml

```
@@ -22,8 +22,9 @@ labels_class:
     ['pattern-block', 'non-pattern-block', 'curb', 'braille-block', 'cement', 'asphalt',
     'cross-walk', 'speed-bump', 'sand', 'bicycle-road', 'deck', 'grass', 'void']
   outdoor_detection:
-    ['person', 'car', 'truck', 'bus', 'motorcycle', 'bicycle', 'kickboard', 'robot', 'mailbox',
-    'walk', 'stop', 'light-out', 'traffic-sign', 'unknown']
+    ['person', 'car', 'truck', 'bus', 'motorcycle', 'bicycle', 'kickboard',
+    'robot', 'animal', 'mailbox', 'traffic-sign', 'unknown',
+    'walk', 'stop', 'light-out', 'countdown-walk', 'countdown-light-out']
```

```suggestion
   'stop',  'walk', 'light-out', 'countdown-walk', 'countdown-light-out']
```
순서 혹시 변경 가능할까요 ㅎㅎ...

---

순서 변경했습니다~!
참고로 리스트를 띄울 때 알파벳 오름차순으로 정렬되어 표기되기 때문에 이 순서에는 딱히 의미가 없습니다.

---

딱히 의미가 없어서 approve 누르고 코멘트를 남겼습니다 ㅎㅎ...
변경 감사합니다 ㅎㅎ